package.json

```diff
@@ -42,5 +42,8 @@
 		"prisma": "^4.1.1",
 		"turbo": "^1.4.2",
 		"typescript": "^4.7.4"
+	},
+	"dependencies": {
+		"@vercel/analytics": "^0.1.8"
```

Comment:
This is the wrong place; `yarn remove @vercel/analytics`, `cd packages/frontend` and reinstall there.